strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/repository/MavenRepository.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.testing.repository;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.carlspring.strongbox.artifact.coordinates.MavenArtifactCoordinates;
+import org.carlspring.strongbox.storage.repository.RepositoryPolicyEnum;
+import org.carlspring.strongbox.testing.storage.repository.RepositorySetup;
+import org.carlspring.strongbox.testing.storage.repository.TestRepository;
+import org.springframework.core.annotation.AliasFor;
+
+/**
+ * @author sbespalov
+ *
+ */
```

Comment:
**Cosmetic:** @sbespalov , would you mind editing your Eclipse style and removing the empty line below the `@author`? Thanks!